src/CMakeLists.txt

```diff
@@ -33,6 +33,12 @@ add_files(
     CONDITION SSE_FOUND
 )
 
+add_files(
+    gfx_layout_icu.cpp
+    gfx_layout_icu.h
+    CONDITION WITH_ICU_LX
```

Comment:
I know it's replaced in following commit, but I'm not sure it's a working condition check at cmake level.

Reply:
It isn't. Will replace with `ICU_lx_FOUND`, which is. But yeah, it is being removed anyway :D